hola_mundo/hola_mundo/urls.py

```diff
@@ -19,3 +19,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
 ]
+
+#Ejemplo de una Rama de Git (ejemplo_rama) esta realizando esta operacion
```

Comment:
Normalmente no semejan comentarios en el codigo (por que el codigo deberia dejarse leer solito)
The code should be self explanatory.



*comment just to leave kind of a review*

Reply:
Esta bien, gracias Reddy! :)